lib/puppet-languageserver/manifest/definition_provider.rb

```diff
@@ -85,14 +85,19 @@ def self.function_name(func_name)
         item = PuppetLanguageServer::PuppetHelper.function(func_name)
         return nil if item.nil? || item.source.nil? || item.line.nil?
         LanguageServer::Location.create(
-          'uri' => 'file:///' + item.source,
+          'uri' => build_uri(item.source),
           'fromline' => item.line,
           'fromchar' => 0,
           'toline' => item.line,
           'tochar' => 1024
         )
       end
       private_class_method :function_name
+
+      def self.build_uri(path)
```

Comment:
I wonder if this should be a generic helper. Although, nothing else is broken yet...

Reply:
I see this being useful in other providers that handle opening or doing things to files